templates/vue/src/components/modals/NodeModal/forms/ContentForm/AnswerForm.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div>
+    <b-form-group data-qa="activity-combobox" label="Select an activity:">

```

Comment:
Although I like the directive language here, to keep consistent with other contents, let's just use "Activity" as the label here (without a colon).